### PR TITLE
[Gradebook] Search fix

### PR DESCRIPTION
**Pull request by otecMuxah**
subj
- [ia-71](https://youtrack.raccoongang.com/issue/IA-71)

### CODE REVIEW

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js [hunk 1]
```diff
@@ -40,27 +36,52 @@ function GradebookTab(button, content) {
...
     }
+    let inputValue = '';
 
     function updateTables() {
         var htmlStringStudents = '';
         var htmlStringResults = '';
+
         var htmlTemp = `<div class="gradebook-table-cell"><form class="student-search">
-            <input type="search" class="student-search-field" placeholder="Search students"/>
+            <input value="${inputValue}" type="search" class="student-search-field" placeholder="Search students"/>
```
Comment (flying-pi) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: Please use Django translations for the placeholder.

**Review by flying-pi — CHANGES_REQUESTED** [review]

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js (hunk 1 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: done

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js [hunk 2]
```diff
@@ -40,27 +36,52 @@ function GradebookTab(button, content) {
...
     }
+    let inputValue = '';
 
     function updateTables() {
         var htmlStringStudents = '';
         var htmlStringResults = '';
+
         var htmlTemp = `<div class="gradebook-table-cell"><form class="student-search">
-            <input type="search" class="student-search-field" placeholder="Search students"/>
+            <input value="${django.gettext(inputValue)}" type="search" class="student-search-field" placeholder="Search students"/>
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: too long string

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js [hunk 3]
```diff
@@ -40,27 +36,52 @@ function GradebookTab(button, content) {
...
+            htmlTemp += `
+                <div class="gradebook-table-cell">
+                    <div class="assignment-label">${greadebookTab.examNames[i]}</div>
+                </div>
+            `;
         }
         greadebookTab.gradebookTableHeader.append(htmlTemp);
+
+        let $input = $('.student-search-field');
+        $input[0].addEventListener('keyup',function(e){
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: spaces after the commas are welcome

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js (hunk 2 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: done

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js (hunk 3 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: done

**Review by idegtiarov — APPROVED** [review]

**Review by flying-pi — APPROVED** [review]

**Pull request merged**